packages/priority-queue/src/index.js

```diff
@@ -0,0 +1,50 @@
+const requestFrame = window.requestIdleCallback ? window.requestIdleCallback : window.requestAnimationFrame;
```

Comment:
Given its name, it seems to imply its some polyfill for `requestAnimationFrame`, whereas in its use we call to the callback parameter as though it can be the one provided by `requestIdleCallback` (the deadline object). For me, I might wonder about inverting this, as a polyfill for `requestIdleCallback` which uses `requestAnimationFrame` as the fallback mechanism.